### PR TITLE
Add python pypi test

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #7048

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 dist: xenial
 language: python
 python: 3.7
+addons:
+  hosts:
+    - pulp
```
Comment (mdellweg) [inline .travis.yml:6]: What is this meant for?
Is there an addon service in travis that provides a pulp server?

Comment (mdellweg) [reply .travis.yml:6]: Found it. ;)
https://docs.travis-ci.com/user/hosts/

Comment (daviddavis) [reply .travis.yml:6]: I wish! This is [to set a custom hostname in travis](https://docs.travis-ci.com/user/hosts/).

Comment (daviddavis) [reply .travis.yml:6]: You beat me. I should probably use a different name to avoid confusion. `pulp-fixtures` maybe?

File: .travis.yml [hunk 2]
```diff
@@ -17,14 +20,15 @@ jobs:
...
-        - curl -L localhost:8000/rpm-unsigned/?badtoken | grep "Wrong auth token"
-        - curl --fail -L localhost:8000/rpm-unsigned/?secret
-        - curl --fail -L localhost:8000/file-large/?parameter
-        - '[[ $(curl localhost:8000/rpm-mirrorlist-good) == "http://example.com/rpm-unsigned/" ]] || exit 1'
+        - curl --fail pulp:8000/file/PULP_MANIFEST
+        - curl --fail pulp:8000/debian/dists/ragnarok/Release
+        - curl -L pulp:8000/rpm-unsigned/?badtoken | grep "Wrong auth token"
+        - curl --fail -L pulp:8000/rpm-unsigned/?secret
+        - curl --fail -L pulp:8000/file-large/?parameter
+        - '[[ $(curl pulp:8000/rpm-mirrorlist-good) == "http://pulp/rpm-unsigned/" ]] || exit 1'
```
Comment (mdellweg) [inline .travis.yml:30]:
```suggestion
        - '[[ $(curl pulp:8000/rpm-mirrorlist-good) == "http://pulp:8000/rpm-unsigned/" ]] || exit 1'
```

File: .travis.yml (hunk 1 above)
Comment (mdellweg) [reply .travis.yml:6]: Yes, good point.

**Pull request merged**